packages/portfolio-planner/src/engine.ts

```diff
@@ -129,9 +132,46 @@ export const startEngine = async ({
   console.warn('Retrieved info for Agoric chain', agoricChainId);
 
   await rpc.opened();
-  try {
-    // console.warn('RPC client opened:', rpc);
+  // console.warn('RPC client opened:', rpc);
+
+  const { fromCapData } = makeClientMarshaller();
 
+  // TODO: replace agd here and handle paginated responses.
+  const portfoliosResp = (
+    await agd.exec(['query', 'vstorage', 'children', VSTORAGE_PATH_PREFIX])
```

Comment:
I thought we were gonna use the vstorage utils that @dckc wired in the other file.

Reply:
To be done after this merges.